File: trail1/A부터 B까지의 합/sum-from-a-to-b.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        var(A, B) = (int.Parse(arr[0]), int.Parse(arr[1]));
        int ssum = 0;
        for (int i=A; i<=B; i++) {
            ssum += i;
        }
        Console.WriteLine(ssum);
    }
}

File: trail1/A부터 B까지의 합/sum-from-a-to-b.c
#include <stdio.h>

int main() {
    int A, B;
    scanf("%d %d", &A, &B);
    int ssum = 0;
    for (int i=A; i<=B; i++) {
        ssum += i;
    }
    printf("%d", ssum);
    return 0;
}